components/Details.js

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from 'react';
 
-import { FormInput } from './Form/FormInput';
+import { FormInput } from './Form/FormInput.tsx';
```

Comment:
I have pulled the branch, it works perfectly fine without it

Reply:
I see Module not found error.
![Snímek obrazovky 2022-06-26 v 16 34 02](https://user-images.githubusercontent.com/65444975/175819453-85b69e42-280e-4558-b3a3-d27a14bf7432.png)

Reply:
But I think there is something strange with the imports.
![Snímek obrazovky 2022-06-26 v 17 05 25](https://user-images.githubusercontent.com/65444975/175820717-991730d4-6cb4-4a9b-8b21-7fed47ddb473.png)

Reply:
It try to find the registration in .next/server/pages